datadog/metric.go

```diff
@@ -94,14 +91,14 @@ func (m *Metric) StackdriverData(ctx context.Context, lastPoint time.Time, rec s
 		return nil, nil, err
 	}
 	if len(series) == 0 {
-		log.Infof(ctx, "Datadog query '%s' returned no time series", m.config.Query)
+		log.WithContext(ctx).Infof("Datadog query '%s' returned no time series", m.config.Query)
```

Comment:
Just FYI, go supports a `%q` token in printf style variables, that will quote the string parameter

```go
log.WithContext(ctx).Infof("Datadog query %q returned no time series", m.config.Query)
```

It's just shorthand.

Reply:
Didn't know about that, thanks! Changed :)